test/unit/replace_version_test.py

```diff
@@ -0,0 +1,29 @@
+import pytest
```

Comment:
you may also want to add a test for `update_versions` (`update_version` vs `update_versions`).